trade-finance-manager-ui/server/controllers/utilisation-reports/add-to-an-existing-payment/get-back-link-href.test.ts

```diff
@@ -0,0 +1,51 @@
+import { getAddToExistingPaymentBackLinkToUtilisationReportReconciliationPage } from './get-back-link-href';
```

Comment:
Can we rename this it's very long, 68 characters 🤯

Reply:
Could it just be `getBackLinkHref`?

Reply:
Renamed to `getLinkToPremiumPaymentsTab`